### PR TITLE
[11.0][MIG] event_registration_cancel_reason: Migration to 11.0

### PR DESCRIPTION
**Pull request by cristinamartinrod**

### CODE REVIEW

File: event_registration_cancel_reason/__init__.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
...
 # Copyright 2016 Antiun Ingeniería S.L.
-# Copyright 2016 Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+# Copyright 2016 Tecnativa - Pedro M. Baeza 
+# Copyright 2017 Tecnativa - Vicent Cubells
+# Copyright 2018 Tecnativa - Cristina Martin
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
 from . import models
-from . import wizard
+from . import wizard
```
Comment (chienandalu) [inline event_registration_cancel_reason/__init__.py]: Missing blank line at EOF

File: event_registration_cancel_reason/README.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (chienandalu) [inline event_registration_cancel_reason/README.rst]: Rebuild README from README fragments locally

File: event_registration_cancel_reason/__init__.py (hunk 1 above)
Comment (pedrobaeza) [inline event_registration_cancel_reason/__init__.py]: Don't remove final line

**Review by chienandalu — CHANGES_REQUESTED** [review]
Tested on runbot :+1: Just check Travis LINT fails and some minor points marked below:

File: event_registration_cancel_reason/models/__init__.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
-# -*- coding: utf-8 -*-
 # Copyright 2016 Antiun Ingeniería S.L.
-# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2016 Tecnativa - Pedro M. Baeza
```
Comment (pedrobaeza) [inline event_registration_cancel_reason/models/__init__.py]: Remove copyright on `__init__` (it's a bad old habit).

File: event_registration_cancel_reason/readme/CREDITS.rst [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Authors
+~~~~~~~
+
+* Tecnativa
+* Antiun Ingeniería S.L.
```
Comment (pedrobaeza) [inline event_registration_cancel_reason/readme/CREDITS.rst]: No need to put this file. Authors are put automatically by the script. This file is intended for extra credits, like funders or so.

File: event_registration_cancel_reason/__manifest__.py [hunk 5]
```diff
@@ -1,18 +1,18 @@
...
+# Copyright 2017 Tecnativa - Vicent Cubells
+# Copyright 2018 Tecnativa - Cristina Martin
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
 {
     "name": "Reasons for event registrations cancellations",
-    "version": "9.0.1.0.0",
+    "version": "11.0.1.0.0",
     "license": "AGPL-3",
     "author": "Antiun Ingeniería S.L., "
```
Comment (pedrobaeza) [inline event_registration_cancel_reason/__manifest__.py]: Remove this company. It's Rafa's old one, and the IP now has been passed to Tecnativa.

File: event_registration_cancel_reason/wizard/__init__.py [hunk 6]
```diff
@@ -1,6 +1,7 @@
-# -*- coding: utf-8 -*-
 # Copyright 2016 Antiun Ingeniería S.L.
-# Copyright 2016 Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+# Copyright 2016 Tecnativa - Pedro M. Baeza
```
Comment (pedrobaeza) [inline event_registration_cancel_reason/wizard/__init__.py]: Remove copyrights.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
You have these flake8 errors:
```
event_registration_cancel_reason/__manifest__.py:2:44: W291 trailing whitespace
event_registration_cancel_reason/models/event_registration.py:2:44: W291 trailing whitespace
event_registration_cancel_reason/tests/test_event_registration_cancel_reason.py:2:44: W291 trailing whitespace
event_registration_cancel_reason/wizard/event_registration_cancel_log_reason.py:2:44: W291 trailing whitespace
```

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@chienandalu you can re-review

**Review by chienandalu — COMMENTED** [review]
@cristinamartinrod You didn't regenerate the README from the new parts.

**Comment by pedrobaeza** [comment]
@chienandalu she is not able to do it right now with Mac. We are dealing with that, and I'll do it when squashing, but meanwhile, let out that requirement.

**Review by chienandalu — COMMENTED** [review]
Ok then :+1:

**Pull request merged**